src/app.js

```diff
@@ -1 +1,32 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { terminal } = require('./terminal');
+const { generate } = require('./generate');
+const { calculateCowsAndBulls } = require('./calculate');
+
+const number = generate();
+const question = 'Enter a 4-digit number: ';
+
+const checkAnswer = (message) => {
+  if (isNaN(+message) || message.length !== 4) {
+    console.log('Invalid enter');
+    terminal.question(question, checkAnswer);
+
+    return;
+  }
```

Comment:
`the program check the entered number (if it has exactly 4 different digits)`

you I don't see this check

Reply:
message.length !== 4 ?

> `the program check the entered number (if it has exactly 4 different digits)`
> 
> you I don't see this check